### PR TITLE
Update prices_ronin_tokens.sql

### PR DESCRIPTION
**Pull request by peterrliem**
Added PIXEL and YGG

## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

[...]

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Hosuke** [comment]
I may need to update the prices checker for ronin...
Let me add something in this PR.

File: dbt_subprojects/tokens/models/prices/ronin/prices_ronin_tokens.sql [hunk 1]
```diff
@@ -23,4 +23,8 @@ FROM
     , ('usdc-usd-coin', 'USDC', 0x0b7007c13325c48911f73a2dad5fa5dcbf808adc, 6)
     , ('axs-axie-infinity', 'AXS', 0x97a9107c1793bc407d6f527b77e7fff4d812bece, 18)
     , ('slp-smooth-love-potion', 'SLP', 0xa8754b9fa15fc18bb59458815510e40a12cd2014, 0)
-) as temp (token_id, symbol, contract_address, decimals)
\ No newline at end of file
+    , ('pixel-pixels', 'PIXEL', 0x7eae20d11ef8c779433eb24503def900b9d28ad7, 18)
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/ronin/prices_ronin_tokens.sql:26]: New prices line without blockchain values are supported in tests in #7142

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @peterrliem

**Review by jeff-dude — APPROVED** [review]

**Comment by peterrliem** [comment]
thank you @Hosuke !

**Pull request merged**